homeassistant/components/zwave_js/translations/en.json

```diff
@@ -51,6 +51,16 @@
             }
         }
     },
+    "device_automation": {
+        "trigger_type": {
+            "event.notification.entry_control": "Sent a notification (Entry Control CC)",
+            "event.notification.notification": "Sent a notification (Notification CC)",
```

Comment:
Any reason we would differentiate between those in config/towards the user? Is it possible that a device supports both?

Reply:
yes, it's possible that a device supports both of these triggers. Should we collapse them into a generic notification trigger?

Reply:
If we can, please. 

Also, is it besides theoretically possible also ever going to occur? Or is one the old one and the other the new one?

Reply:
that's a better question for @AlCalzone, I'm not sure

Reply:
oh, the reason we have to have them separately is because they have different schemas. We could probably collapse the schemas but I think that would introduce unnecessary complexity. Thoughts?

Reply:
let's keep it separate. We only show them if the device actually supports them anyway right ?

Reply:
Yes that's correct